apps/etterlatte-trygdetid/src/main/resources/db/migration/V2__legg_til_trygdetid_i_trygdetidgrunnlag.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE trygdetid_grunnlag ADD COLUMN trygdetid BIGINT;
```

Comment:
Hvorfor `BIGINT`? Selv `Integer` er vel alt for stort med sine 4 bytes som gir en maksverdi på 2.147.483.647

https://www.postgresql.org/docs/current/datatype-numeric.html